src/style.css

```diff
@@ -0,0 +1,120 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+}
+
+main {
+  height: 400px;
+  width: 70%;
+  margin: auto;
+  background-color: rgb(234, 232, 232);
+}
+
+.leader-area {
+  display: flex;
+  margin: auto;
+  width: 90%;
+}
+
+h1 {
+  text-align: center;
+  margin-top: 70px;
+  margin-bottom: 55px;
+  padding-top: 20px;
+  font-family: 'Franklin Gothic Medium', 'Arial Narrow', Arial, sans-serif;
+}
+
+.title-and-refresh {
+  display: flex;
+  margin-bottom: 20px;
+  gap: 3%;
+}
+
+.recent-score {
+  font-family: 'Franklin Gothic Medium', 'Arial Narrow', Arial, sans-serif;
+  font-size: 22px;
+}
+
+/* .refresh {
+  padding: 7px;
+} */
+
+.all-score {
+  border: 2px solid black;
+  width: 70%;
+  list-style-type: none;
+  padding: 0;
+}
+
+input {
+  padding: 7px;
+  margin: 10px 10px 10px 0;
+  outline: none;
+}
+
+.each-entry {
+  margin: 5px;
+}
+
+h3 {
+  margin-bottom: 10px;
+  font-family: 'Franklin Gothic Medium', 'Arial Narrow', Arial, sans-serif;
+}
+
+.display-score {
+  margin-left: 15%;
+  width: 100%;
+}
+
+.add-score {
+  width: 100%;
+}
+
+.submit-score-button-div {
+  display: flex;
+  flex-direction: row;
+  align-self: flex-end;
+  justify-content: right;
+  margin-right: 50%;
+}
+
+.submit-score-button {
+  padding: 5px;
+}
+
```

Comment:
- [OPTIONAL] Please it will be nice to align your `Refresh and Submit buttons` to the right of `Recent Score and Add your score` sections respectively as specified in the [wireframe](https://github.com/microverseinc/curriculum-javascript/blob/main/leaderboard/images/leaderboard_wireframe.png).